src/Native/HtmlAsJson.js

```diff
@@ -9,15 +9,18 @@ var _eeue56$elm_html_test$Native_HtmlAsJson = (function() {
         return vNode;
     }
 
-    // stringify needed to strip functions - can performance-optimize this later!
     return {
-        toJsonString: function(html) {
-            var asString = JSON.stringify(forceThunks(html));
+        toJson: function(html) {
+            return forceThunks(html);
+        },
+        getEventDecoder: F2(function (name, events) {
+            var event = events[name];
+            if (!event) return { ctor: 'Nothing' };
```

Comment:
Better to make this an error: https://github.com/eeue56/elm-server-side-renderer/blob/master/src/Native/ServerSideHelpers.js#L93